.github/workflows/release-tags.yml

```diff
@@ -76,13 +76,20 @@ jobs:
         continue-on-error: true
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        if: ${{ !contains(github.event.head_commit.message, '[no github]') }}
```

Comment:
maybe switch this to [snapshot] and also apply it to the MN/CF section below?  Or make it a global variable MVN_ONLY.

Reply:
Sounds reasonable